### ISSUE TITLE
SGD Learning Rate 'Burn-In'

### ISSUE THREAD

**Issue by mvanlierBCG** [david8862/keras-YOLOv3-model-set#40]
Is there an option (currently), or the possibility to implement Darknet 'Burn-In' phase?

This question is partially a copy of [https://github.com/ultralytics/yolov3/issues/15]

**Comment by david8862** [comment]
there's an comment out keras `learning_rate_scheduler()` implementation [here](https://github.com/david8862/keras-YOLOv3-model-set/blob/master/common/model_utils.py#L48). You can change it for the 'Burn-In' phase trick and apply to the training process.

**Comment by mvanlierBCG** [comment]
thankyou, that is clear.
